scripts/release/create_minor_release_PR_commit.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+set -eu
+
+# Folder, where we have cloned repositories' sources
+SOURCES_DIR="sources"
+
+CHANGELOG_HEADER_LINES="6"
+
+GH_USERNAME=$(gh auth status | grep 'Logged in to' |cut -d ' ' -f 9)
+
+FORK_DIR="$GH_USERNAME-timescaledb"
+
+echo "---- Deriving the release related versions from main ----"
+
+cd ~/"$SOURCES_DIR"/"$FORK_DIR"
+git fetch --all
+
+NEW_PATCH_VERSION="0"
+NEW_VERSION=$(head -1 version.config | cut -d ' ' -f 3 | cut -d '-' -f 1)
+RELEASE_BRANCH=$(echo "${NEW_VERSION/%.$NEW_PATCH_VERSION/.x}")
+CURRENT_VERSION=$(tail -1 version.config | cut -d ' ' -f 3)
+cd sql/updates
+LAST_VERSION=$(ls |grep $CURRENT_VERSION.sql |cut -d '-' -f 1)
+
+echo "CURRENT_VERSION is $CURRENT_VERSION"
+echo "LAST_VERSION is $LAST_VERSION"
+echo "RELEASE_BRANCH is $RELEASE_BRANCH"
+echo "NEW_VERSION is $NEW_VERSION"
+cd ~/"$SOURCES_DIR"/"$FORK_DIR"
+
+
+# Derived Variables
+RELEASE_PR_BRANCH="release-$NEW_VERSION"
+UPDATE_FILE="$CURRENT_VERSION--$NEW_VERSION.sql"
+DOWNGRADE_FILE="$NEW_VERSION--$CURRENT_VERSION.sql"
+LAST_UPDATE_FILE="$LAST_VERSION--$CURRENT_VERSION.sql"
+LAST_DOWNGRADE_FILE="$CURRENT_VERSION--$LAST_VERSION.sql"
+
+
+echo "---- Creating release branch $RELEASE_PR_BRANCH from $RELEASE_BRANCH, on the fork ----"
+
+git checkout -b "$RELEASE_PR_BRANCH" upstream/"$RELEASE_BRANCH"
+git branch
+git pull && git diff HEAD
+
+
+echo "---- Modifying version.config to the new versions ----"
+
+sed -i.bak "s/"-dev"/""/g" version.config
```

Comment:
do you need those extra quotes?
```suggestion
sed -i.bak "s/-dev//g" version.config
```

Reply:
Could  also use `'` instead if your shell chokes on those

Reply:
alright, modified so.